packages/ladot-service-areas/ladot-service-areas.js

```diff
@@ -3,8 +3,6 @@ const restrictedAreas = require('./restricted-areas')
 const laDacs = require('./la-dacs')
 const veniceSpecOps = require('./venice-special-ops-zone')
 
-// FIXME move into DB eventually
```

Comment:
Removed because we're deprecating service areas SoonTM?

Reply:
Yah!